### PR TITLE
Hero title changed

### PR DESCRIPTION
**Pull request by ImIsrafil**
Changes:
-Hero or Page title is changed

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
     <img id="banner" src="banner.jpg" alt="Banner" height="240">
 
     <h1>
-        About to Jovian
+        Welcome to Jovian
```
Comment (astro-israfil) [inline src/index.html]: Maybe this title should be more introducing.

**Review by astro-israfil — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (ImIsrafil) [reply src/index.html]: Ok I am doing the change right now

File: src/index.html [hunk 2]
```diff
@@ -30,7 +30,7 @@
     <img id="banner" src="banner.jpg" alt="Banner" height="240">
 
     <h1>
-        About to Jovian
+        Welcome to Jovian to build your bright career
```
Comment (ImIsrafil) [inline src/index.html:33]: That's the changes I made. is it ok or not?

Comment (astro-israfil) [reply src/index.html:33]: Oh, that's now looks great

**Review by astro-israfil — APPROVED** [review]

**Pull request merged**